specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -275,10 +275,6 @@
             }
           }
         },
-        "x-ms-long-running-operation": true,
```

Comment:
You can't make changes to existing api-versions.  If you feel these are necessary please get signoff by going through the breaking change review process in the PR description

Reply:
This breaking change has been approved [as part of another PR](https://github.com/Azure/azure-rest-api-specs/pull/12547).

I have pulled latest commits from master to remove it from this pull request.